### PR TITLE
COMMERCE-2216 Speedwell fix initialization

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by gianmarcobrunialti** [comment]
ci:test

File: commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+out
+build
+classes
```
Comment (ethanbustad) [inline commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/.gitignore]: I don't think we should need to add this file.... AFAIK, `build` and `classes` should already be ignored, but maybe `out` isn't? What's writing to `out`?

Comment (gianmarcobrunialti) [reply commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/.gitignore]: @ethanbustad I saw many `out` directory being listed and available for git staging... I believe it's FE stuff, but didn't make sure, I was just remembering that, by the book, we have to git-ignore that directory.

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
tests should be fixed in upstream now

File: commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/.gitignore (hunk 1 above)
Comment (ethanbustad) [reply commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/.gitignore]: @gianmarcobrunialti alright, we can leave it in for now. There are some tests that ensure the files we include match what's expected, and we can just wait and see if there are any problems stemming from this change, and only fix when necessary.

**Review by ethanbustad — APPROVED** [review]

**Pull request merged**